src/controller/model/device.ts

```diff
@@ -406,6 +406,16 @@ class Device extends Entity {
             return true;
         }
 
+        // Similar to TuYa devices, Third Reality devices don't change zoneState after enroll
+        if this.modelID?.match('^3R.*?Z') {
```

Comment:
Instead of adding here, can you try adding it to the lookup below (line 424):

```
'^3R.*?Z': {type: 'EndDevice', powerSource: 'Battery'}
```

Reply:
Yep, that works. Thanks! I figured what I did probably wasn't ideal lol

Reply:
Can you update this pr with that change?

Reply:
Done!